README.md

```diff
@@ -1,4 +1,6 @@
-# netdata [![Build Status](https://travis-ci.com/netdata/netdata.svg?branch=master)](https://travis-ci.com/netdata/netdata) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2231/badge)](https://bestpractices.coreinfrastructure.org/projects/2231) [![License: GPL v3+](https://img.shields.io/badge/License-GPL%20v3%2B-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+# netdata README
```

Comment:
This information is redundant and puts emphasis on README part instead of netdata part. Look how others are doing this, ex: https://github.com/etcd-io/etcd or https://github.com/kubernetes/kubernetes  or even https://github.com/prometheus/prometheus

Reply:
I am not sure I follow.

How is the existing different from this one:

![image](https://user-images.githubusercontent.com/2662304/46251391-30d27600-c45a-11e8-9ea3-18c8f6dd9d78.png)

because they have 1 badge and we have 3?

Also I don't see any reason to put emphasis on README instead of netdata.

Reply:
Now got it. @paulfantom agrees with me. Sorry.